src/main/drivers/bus_i2c.h

```diff
@@ -19,22 +19,15 @@
 
 #include "platform.h"
 
-#include "pg/pg.h"
 #include "drivers/io_types.h"
 #include "drivers/rcc_types.h"
 
+#include "pg/bus_i2c.h"
```

Comment:
Device driver `.h` files should not include parameter group `.h` files, see rules discussed in https://github.com/betaflight/betaflight/pull/4679

Reply:
This is a bit of a conundrum then, since `enum I2CDevice` is used in both  `pg/bus_i2c.c` and `drivers/bus_i2c_common.c`, and we don't allow forward declarations of enums. What do we suggest we do.

Reply:
Keep `enum I2CDevice` in `bus_i2c.h` and include `bus_i2c.h` where necessary.

Reply:
Which `bus_i2c`? : -)

(Doing this makes me wonder if there is a better way of naming these, to get avoid creating a large number of file name clashes.)

Reply:
Keep enum I2CDevice in `drivers/bus_i2c.h` and include `drivers/bus_i2c.h` where necessary.